docs/develop/excel-async-model.md

```diff
@@ -0,0 +1,180 @@
+---
+title: 'Using the async Office Scripts APIs in performance-critical scenarios'
+description: 'A primer on the Office Scripts async APIs and how to use the load/sync pattern to maximize script performance.'
+ms.date: 05/22/2020
+localization_priority: Normal
+---
+
+
+# Using the async Office Scripts APIs in performance-critical scenarios
+
+This article will teach you how to write scripts using the async APIs. These APIs let your script control the data synchronization between the script and the workbook. This gives you maximum control over the network calls to and from the workbook, which are the primary cause of performance issues.
+
+> [!IMPORTANT]
+> The async model is significantly more complicated than the standard Office Scripts APIs. We highly recommend following the guidance in [Improve the performance of your Office Scripts](web-client-performance.md) before switching to the async APIs.
```

Comment:
Given the situation with this, can you say we recommend using this only if you have performance related concerns or have already been using code that does this? Otherwise, the other one

Reply:
Current usage is pretty low, so I wanted to frame everything to be forward-facing.